Components/RTShaderSystem/src/OgreShaderProgramManager.cpp

```diff
@@ -294,6 +293,12 @@ GpuProgramPtr ProgramManager::createGpuProgram(Program* shaderProgram,
             programName, ResourceGroupManager::INTERNAL_RESOURCE_GROUP_NAME);
 
     if(pGpuProgram) {
+        if (!cachePath.empty())
+            LogManager::getSingleton().logMessage(shaderProgram->getEntryPointFunction()->getDescription()
+                + ' ' + to_string(shaderProgram->getType())
```

Comment:
after a closer look, this is actually redundant with `programName` as the latter is suffixed wit _VS/ _FS

Reply:
As a material may use an existing cache entry, this is needed to get all the material-to-shader mappings.